### PR TITLE
Add Cotx Hotspot

### PR DESCRIPTION
**Pull request by cotxliang**

### CODE REVIEW

**Comment by cotxliang** [comment]
@madninja I've added Cotx hotspots to the helium app. Is there anything else I need to add？

**Comment by madninja** [comment]
> @madninja I've added Cotx hotspots to the helium app. Is there anything else I need to add？

Not my PR to review..

File: src/makers/cotx/hotspots.ts
Comment (cokes518) [inline src/makers/cotx/hotspots.ts:11]: Please replace `Helium` with your company name.

Comment (cokes518) [reply src/makers/cotx/hotspots.ts:11]: And use your support email here

File: src/makers/cotx/hotspots.ts
Comment (cokes518) [inline src/makers/cotx/hotspots.ts:23]: same thing here. Even though it's in Japanese, you can easily replace Helium and the Helium support email with your own company name and support email.

**Review by cokes518 — CHANGES_REQUESTED** [review]

File: src/makers/cotx/hotspots.ts
Comment (cotxliang) [reply src/makers/cotx/hotspots.ts]: > same thing here. Even though it's in Japanese, you can easily replace Helium and the Helium support email with your own company name and support email.

Ok, I've updated this questions.

Comment (cotxliang) [reply src/makers/cotx/hotspots.ts]: > same thing here. Even though it's in Japanese, you can easily replace Helium and the Helium support email with your own company name and support email.

Hi, I have submitted the code again. Please review it when you have time.

File: src/makers/cotx/hotspots.ts
Comment (cokes518) [inline src/makers/cotx/hotspots.ts:14]:
```suggestion
        'Once the COTX Gateway Miner is powered on, the display screen will appear.',
```

File: src/makers/cotx/hotspots.ts
Comment (cokes518) [inline src/makers/cotx/hotspots.ts:18]:
```suggestion
        'The Hotspot is ready to pair when the screen displays the Bluetooth icon.\n\nMake sure your phone’s Bluetooth is turned on!',
```

Comment (cotxliang) [reply src/makers/cotx/hotspots.ts]: > 

Thank you. I have corrected the problem again. Please let me know if any changes are needed.

**Review by cokes518 — APPROVED** [review]
this is good to merge as well cc @tyler-whitman

**Review by tyler-whitman — APPROVED** [review]

File: src/makers/cotx/index.ts
Comment (tyler-whitman) [inline src/makers/cotx/index.ts:7]: @cokes518 I dont see this in the onboarding server yet, has it been approved?

Comment (cotxliang) [reply src/makers/cotx/index.ts:7]: > @cokes518 I dont see this in the onboarding server yet, has it been approved?

This id number is added by us according to the previously approved company id sequence. I am not sure how much id is written. Does the id number need to be approved by the onboarding server before it can be written here?

**Comment by cokes518** [comment]
Hi @cotxliang as we await the MOC approval, please let me know the name you'd like to appear in the Onboarding server, and how many location asserts you'd like to include (minimum is 1).

Here is an example of other Makers and the names they use.
https://onboarding.dewi.org/api/v2/makers

File: src/makers/cotx/index.ts
Comment (tyler-whitman) [inline src/makers/cotx/index.ts:7]: wait for official onboarding id

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

**Comment by cokes518** [comment]
> Hi @cotxliang as we await the MOC approval, please let me know the name you'd like to appear in the Onboarding server, and how many location asserts you'd like to include (minimum is 1).
> 
> Here is an example of other Makers and the names they use.
> https://onboarding.dewi.org/api/v2/makers

@cotxliang we need a response or else we won't be able to create the account.

**Comment by cotxliang** [comment]
> > Hi @cotxliang as we await the MOC approval, please let me know the name you'd like to appear in the Onboarding server, and how many location asserts you'd like to include (minimum is 1).
> > Here is an example of other Makers and the names they use.
> > https://onboarding.dewi.org/api/v2/makers
> 
> @cotxliang we need a response or else we won't be able to create the account.

hi @cokes518 
name : COTX Networks
location limit : 1

**Comment by cokes518** [comment]
maker account created.

```{
    "code": 200,
    "data": {
        "id": 16,
        "name": "COTX Networks",
        "address": "13cbbZXzqwp6YMM5JvAu5T1TRhenENEJVU5Q8vpLhunQYE1Acpp",
        "locationNonceLimit": 1
    },
    "success": true
}```

@cotxliang please update the pull request to use maker id 16.

**Comment by cotxliang** [comment]
> maker account created.
> 
> ```
> {
>     "code": 200,
>     "data": {
>         "id": 16,
>         "name": "COTX Networks",
>         "address": "13cbbZXzqwp6YMM5JvAu5T1TRhenENEJVU5Q8vpLhunQYE1Acpp",
>         "locationNonceLimit": 1
>     },
>     "success": true
> }
> ```
> 
> @cotxliang please update the pull request to use maker id 16.

maker id updated.

**Comment by cokes518** [comment]
@tyler-whitman over to you. should be good to merge if you're okay with it.

File: src/makers/cotx/hotspots.ts
Comment (tyler-whitman) [inline src/makers/cotx/hotspots.ts:6]: You have some lint errors in this file you need to fix. You can see them by running `yarn lint` in the root folder:
```
  30:59  error  Replace `\·n·\` with `·n·`                                                                                                                                                     prettier/prettier
  30:59  error  Unnecessary escape character: \ .                                                                                                                                              no-useless-escape
  30:63  error  Unnecessary escape character: \ .                                                                                                                                              no-useless-escape
  59:13  error  Replace `·us:·ANTENNAS.COTX_US,·default:·ANTENNAS.COTX_EU,·cn:·ANTENNAS.COTX_CN` with `⏎····us:·ANTENNAS.COTX_US,⏎····default:·ANTENNAS.COTX_EU,⏎····cn:·ANTENNAS.COTX_CN,⏎·`  prettier/prettier

✖ 4 problems (4 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

**Review by tyler-whitman — CHANGES_REQUESTED** [review]
There are some lint errors and a merge conflict to fix and this will be good to merge.

File: src/makers/cotx/hotspots.ts
Comment (cotxliang) [reply src/makers/cotx/hotspots.ts:6]: > You have some lint errors in this file you need to fix. You can see them by running `yarn lint` in the root folder:
> 
> ```
>   30:59  error  Replace `\·n·\` with `·n·`                                                                                                                                                     prettier/prettier
>   30:59  error  Unnecessary escape character: \ .                                                                                                                                              no-useless-escape
>   30:63  error  Unnecessary escape character: \ .                                                                                                                                              no-useless-escape
>   59:13  error  Replace `·us:·ANTENNAS.COTX_US,·default:·ANTENNAS.COTX_EU,·cn:·ANTENNAS.COTX_CN` with `⏎····us:·ANTENNAS.COTX_US,⏎····default:·ANTENNAS.COTX_EU,⏎····cn:·ANTENNAS.COTX_CN,⏎·`  prettier/prettier
> 
> ✖ 4 problems (4 errors, 0 warnings)
>   2 errors and 0 warnings potentially fixable with the `--fix` option.
> ```

The above questions have been revised.

Comment (tyler-whitman) [reply src/makers/cotx/hotspots.ts:6]: @cotxliang Thanks for the lint fixes there is still a merge conflict to resolve

Comment (cotxliang) [reply src/makers/cotx/hotspots.ts:6]: Thank you. I just corrected the question. Do you think it’s okay?

File: src/makers/cotx/hotspots.ts
Comment (tyler-whitman) [inline src/makers/cotx/hotspots.ts:62]: This is not a valid property. It needs to be removed, only `us` and `default` are valid which you can see when running `yarn lint`. 

The way it currently works is that if the user is in the US it picks the `antenna.us` else it will pick the `antenna.default`. So you can either default that to `ANTENNAS.COTX_EU` or `ANTENNAS.COTX_CN`.

The user can then edit it if it's incorrect.

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

File: src/makers/cotx/hotspots.ts
Comment (cotxliang) [reply src/makers/cotx/hotspots.ts]: @tyler-whitman Done

Comment (cotxliang) [reply src/makers/cotx/hotspots.ts]: @tyler-whitman  Done

Comment (cotxliang) [reply src/makers/cotx/hotspots.ts]: I updated this question, please review, thank you.

Comment (cotxliang) [reply src/makers/cotx/hotspots.ts]: @tyler-whitman I updated this question, please review, thank you

**Review by tyler-whitman — APPROVED** [review]

**Pull request merged**
